src/calendar/index.njk

```diff
@@ -2,44 +2,121 @@
 title: Calendar
 layout: layouts/base.njk
 ---
-<div class="grid gap-4">
-  <h2>Events this week</h2>
 
+<div class="grid grid-flow-row gap-3">
```

Comment:
I don't think you need the `grid-flow-row` here, can `gap` be set to `4` to keep consistency with the rest of the site please?